mitodl/ocw-studio
# node-sass -> sass

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

this just removes `node-sass` in favor of `sass`. `node-sass` is a node package that incorporates the [now deprecated](https://sass-lang.com/blog/libsass-is-deprecated) `libsass` C library, which means that using it in a JS environment is always kind of a pain (as native code in JS tends to be).

Anyhow, sass has now been re-written in Dart, which...wouldn't have been my first choice but compiles down to pure JS, so they now can distribute a pure JS library (`sass`) . I tried things out quickly and it seems to be a drop-in replacement.

#### How should this be manually tested?

make sure the styling isn't weird anywhere, build works properly, everything like that. also enjoy a brief island of sanity in the whirlwind that is the JS ecosystem.
